scripts/setup_webserver.sh

```diff
@@ -87,6 +87,9 @@ check_fileServerType_param $fileServerType
   sudo apt-get install -y graphviz aspell php-soap php-json php-redis php-bcmath php-gd php-pgsql php-mysql php-xmlrpc php-intl php-xml php-bz2
   install_php_sql_driver
 
+  # PHP Version
+  PhpVer=`/usr/bin/php -r "echo PHP_VERSION;" | /usr/bin/cut -c 1,2,3`
```

Comment:
As I mentioned earlier, this logic shouldn't be repeated here, but refactored to a function and placed in the `helper_functions.sh`, even though it's just a one-liner.